win32ss/user/ntuser/winsta.h

```diff
@@ -11,18 +11,16 @@
 #define WSS_DYING         (16)
 #define WSS_REALSHUTDOWN  (32)
 
+// See also: https://reactos.org/wiki/Techwiki:Win32k/WINDOWSTATION
 typedef struct _WINSTATION_OBJECT
 {
     DWORD dwSessionId;
 
     LIST_ENTRY DesktopListHead;
     PRTL_ATOM_TABLE AtomTable;
-    HANDLE ShellWindow;
-    HANDLE ShellListView;
-
-    ULONG Flags;
-    struct _DESKTOP* ActiveDesktop;
 
+    ULONG          Flags;
+    struct tagKL  *spklList;
```

Comment:
```suggestion
    struct tagKL*  spklList;
```

Reply:
We cannot use "PKL" as type directly here?

Reply:
No we can't.

Reply:
OK that's certainly like some of these other structures (like desktop etc) being used there.